src/main/webapp/search-script.js

```diff
@@ -69,6 +69,7 @@ async function addCourse() {
   url.searchParams.set("num-units", units);
   url.searchParams.set("term", termName);
   url.searchParams.set("school-name", schoolName);
+  url.searchParams.set("num-enrolled", "300");
```

Comment:
300 seems kinda arbitrary? Is this for mock data or testing or something?

Reply:
Yes that is temporary will be removed in future PR. We just need to be passing in something for the time being so it does not 500 multiple servlets.